README.md

```diff
@@ -556,6 +564,9 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**SuperFreezZ**](https://gitlab.com/SuperFreezZ/SuperFreezZ) <sup>**[[F-Droid](https://f-droid.org/app/superfreeze.tool.android)]**</sup>
 * [**Battery Tool**](https://github.com/Domi04151309/BatteryTool) <sup>**[[F-Droid](https://f-droid.org/app/io.github.domi04151309.batterytool)]**</sup>
 * [**AppManager**](https://github.com/MuntashirAkon/AppManager) <sup>**[[F-Droid](https://f-droid.org/app/io.github.muntashirakon.AppManager)]**</sup>
+* [**Inure App Manager**](https://github.com/Hamza417/Inure)
```

Comment:
According to [this discussion](https://github.com/Hamza417/Inure/issues/11) all proprietary parts are removed :+1:.

Reply:
Change app name to "Inure App Manager (Trial)"
Also live on F-Droid: https://f-droid.org/en/packages/app.simple.inure/

Reply:
Updated, thanks!

I don't find much information what _Trial_ version actually does. What happens after the 15 days?

Reply:
IDK never tried it as most of the premium features of Inure App Manager are free in [AM](https://github.com/MuntashirAkon/AppManager) + active development.

Reply:
https://hamza417.gumroad.com/l/inure_unlocker/